### DISCUSSION TITLE
String Delimiter Poll

### DISCUSSION THREAD

**Discussion by lyxal** [Vyxal/Vyxal#745] [Version 3]
What character should be used for strings? I'll post the options as replies to this thread, and you can vote for the one you want using the upvote button that github discussions provide.

Please don't post any new top-level replies/answers to this thread if they aren't suggestions (i.e comment suggestions I haven't already covered, not general discussion). Replies to the options are okay though.

Voting closes UTC midnight on Sunday.

**Comment by lyxal** [comment]
# Single Quotation Mark

```
'this is a string'
```

**Comment by lyxal** [comment]
# Double Quotation Mark

```
"this is a string"
```

**Comment by lyxal** [comment]
# Backtick

```
`this is a string`
```

**Comment by lyxal** [comment]
# Left-Pointing Double Angle Quotation Mark

```
«this is a string«
```

**Comment by lyxal** [comment]
# Right-Pointing Double Angle Quotation Mark

```
»this is a string»
```

**Comment by lyxal** [comment]
# Left + Right Pointing Double Angle Quotation Mark

```
«this is a string»
```

**Comment by pxeger** [comment]
# Right + Left Pointing Double Angle Quotation Mark

```
»this is a string«
```
